aperturedb/sqlc/migrations/000003_lnc_sessions.up.sql

```diff
@@ -0,0 +1,29 @@
+-- lnc_sessions is table used to store data about LNC sesssions.
+CREATE TABLE IF NOT EXISTS lnc_sessions (
+    id INTEGER PRIMARY KEY,
+
+    -- The entropy bytes to be used for mask the local ephemeral key during the 
+    -- first step of the Noise XX handshake.
+    passphrase_entropy BLOB NOT NULL UNIQUE,
+
```

Comment:
do we need to store this? we get it from the config file right? and then if the connection works then we dont need this again

Reply:
ah ok yeah you are using it as an identifier. 

so I guess then my question is, why would we need more than 1 session?

Reply:
I think so this can serve as the baiss for other projects/packages? So then they can have multiple sessions themselves, say a single `lncli` instance that can connect to all the other active sessions.

Reply:
We should not need more than one session, but we may need to update it because the old one expired, or we want to use a session with  more constraints for example. Right now there is no "session management" exposed to the user. We basically say "give me the passphrase that you want to use and i will use that to connect to the node"

In the example roasbeef is talking about it makes sense to use labels (so the user can connect to multiple nodes changing a short string in the params) but here I think the trade offs are different enough two  do not follow the same pattern in both cases.